src/components/[guild]/RolePlatforms/components/EditRolePlatformCapacityTimeModal.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Stack,
+  Text,
+  Tooltip,
+} from "@chakra-ui/react"
+import Button from "components/common/Button"
+import CheckboxColorCard from "components/common/CheckboxColorCard"
+import { Modal } from "components/common/Modal"
+import { Clock, Hash } from "phosphor-react"
+import { useController, useForm, useWatch } from "react-hook-form"
+import { PlatformName } from "types"
+
+type RolePlatformCapacityTimeForm = {
+  capacity?: number
+  startTime?: string
+  endTime?: string
+}
+
+type Props = {
+  isOpen: boolean
+  onClose: () => void
+  platformType: PlatformName
+  defaultValues?: RolePlatformCapacityTimeForm
+  isLoading?: boolean
+  onDone: (data: RolePlatformCapacityTimeForm) => void
+}
+
+const AUTO_SUPPLY_PLATFORMS: PlatformName[] = ["UNIQUE_TEXT"]
+
+export const DAY_IN_MS = 1000 * 60 * 60 * 24
+
+const getShortDate = (isoDate: string): string | undefined => {
+  if (!isoDate) return undefined
+  return isoDate.split(/:\d{2}\.\d{3}Z/)[0]
+}
+
+const datetimeLocalToIsoString = (datetimeLocal: string): string | undefined => {
+  if (!datetimeLocal) return undefined
+
+  try {
+    return new Date(datetimeLocal).toISOString()
+  } catch {
+    return undefined
+  }
+}
+
+const EditRolePlatformCapacityTimeModal = ({
```

Comment:
Would be nice to use Availibilty instead of CapacityTime. Not just here but everywhere, except CAPACITY_TIME_PLATFORMS maybe and when we specifically refer to either capacity or time ofc. Availibilty is a nice collective term for them imo